src/cf.data.pre

```diff
@@ -4886,6 +4886,51 @@ DOC_START
 	rotated with "debug_options"
 DOC_END
 
+NAME: cache_log_message
+TYPE: cache_log_message
+DEFAULT: none
+DEFAULT_DOC: Use debug_options.
+LOC: TheDebugMessages
+DOC_START
+	Configures logging of individual cache.log messages.
+
+		cache_log_message id=<number> [option...]
+		cache_log_message ids=<number>-<number> [option...]
+
+	Most messages have _not_ been instrumented to support this directive
+	yet. For the list of instrumented messages and their IDs, please see
+	the doc/debug-sections.txt file.
```

Comment:
Probably we need to write 'debug-messages.txt' here?

Reply:
Agreed.